src/Compilers/CSharp/Test/Emit2/Attributes/AttributeTests_LifetimeAnnotation.cs

```diff
@@ -130,34 +121,58 @@ public void ExplicitAttribute_ReferencedInSource_02()
             var source =
 @"using System;
 using System.Runtime.CompilerServices;
-[module: LifetimeAnnotation(false, true)]
-[LifetimeAnnotation(false, true)] class Program
+[module: ScopedRef]
+[ScopedRef] class Program
 {
-    [LifetimeAnnotation(false, true)] object F;
-    [LifetimeAnnotation(false, true)] event EventHandler E;
-    [LifetimeAnnotation(false, true)] object P { get; }
-    [LifetimeAnnotation(false, true)] static object M1() => throw null;
-    [return: LifetimeAnnotation(false, true)] static object M2() => throw null;
-    static void M3<[LifetimeAnnotation(false, true)] T>() { }
+    [ScopedRef] object F;
+    [ScopedRef] event EventHandler E;
+    [ScopedRef] object P { get; }
+    [ScopedRef] static object M1() => throw null;
+    [ScopedRef] static object M2() => throw null;
+    static void M3<[ScopedRef] T>() { }
 }";
-            var comp = CreateCompilation(new[] { LifetimeAnnotationAttributeDefinition, source });
-            // https://github.com/dotnet/roslyn/issues/62124: Re-enable check for LifetimeAnnotationAttribute in ReportExplicitUseOfReservedAttributes.
+            var comp = CreateCompilation(new[] { ScopedRefAttributeDefinition, source });
+            // https://github.com/dotnet/roslyn/issues/62124: Re-enable check for ScopedRefAttribute in ReportExplicitUseOfReservedAttributes.
             comp.VerifyDiagnostics(
-                // (6,46): warning CS0169: The field 'Program.F' is never used
-                //     [LifetimeAnnotation(false, true)] object F;
-                Diagnostic(ErrorCode.WRN_UnreferencedField, "F").WithArguments("Program.F").WithLocation(6, 46),
-                // (7,58): warning CS0067: The event 'Program.E' is never used
-                //     [LifetimeAnnotation(false, true)] event EventHandler E;
-                Diagnostic(ErrorCode.WRN_UnreferencedEvent, "E").WithArguments("Program.E").WithLocation(7, 58));
+                // (3,10): error CS0592: Attribute 'ScopedRef' is not valid on this declaration type. It is only valid on 'parameter' declarations.
+                // [module: ScopedRef]
+                Diagnostic(ErrorCode.ERR_AttributeOnBadSymbolType, "ScopedRef").WithArguments("ScopedRef", "parameter").WithLocation(3, 10),
```

Comment:
>Diagnostic(ErrorCode.ERR_AttributeOnBadSymbolType, "ScopedRef").WithArguments("ScopedRef", "parameter").WithLocation(3, 10), [](http://example.com/codeflow?start=16&length=124)

Ideally, this test should use a definition of `ScopedRefAttributes` with `AttributeTargets.All` rather than `AttributeTargets.Parameter`, so we can ensure we're reporting errors for the use in source (when [https://github.com/dotnet/roslyn/issues/62124](https://github.com/dotnet/roslyn/issues/62124) is fixed) rather than for incorrect target. 

It's not blocking though.